lib/tasks/application.rake

```diff
@@ -121,4 +123,45 @@ namespace :application do
     Rails.logger = ActiveSupport::Logger.new($stdout)
     Rails.logger.level = 1
   end
+
+  namespace :links_valid do
+    desc "Checks the validity of links in division summary"
+    task divisions: :environment do
+      md = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+
+      Division.find_each do |division|
+        wiki_motion = division.wiki_motion
+
+        if wiki_motion
+          summary_in_html = md.render(wiki_motion.description)
+          parsed_data = Nokogiri::HTML.parse(summary_in_html)
+          url_hash_table = Hash.new(0)
+          broken_urls = []
+
+          tags = parsed_data.xpath("//a")
+          tags.each do |tag|
+            url_from_page = tag[:href]
+            url = url_from_page
```

Comment:
why are you assigning this to `url_from_page` first and then `url`?

Reply:
`url` is used when making the web request as it will always have the `http://` Infront of it even when `url_from_page` doesnt

`url_from_page` will not always have the `http://`. Additionally, its used when printing the broken URL out. You can simply copy and ctrl + f to search for the broken URL (since it does not change but `url` does)

Reply:
That makes sense thanks.